prepare_thomaswue.sh

```diff
@@ -17,3 +17,9 @@
 
 source "$HOME/.sdkman/bin/sdkman-init.sh"
 sdk use java 21.0.1-graal 1>&2
+
+# ./mvnw clean verify removes target/ and will re-trigger native image creation.
+if [ ! -f target/CalculateAverage_thomaswue_image ]; then
```

Comment:
Not sure we want this, what if there was a code change since the last run?

Reply:
`# ./mvnw clean verify removes target/ and will re-trigger native image creation.`

Reply:
Oh, didn't notice you moved the output to the target directory